stdlib/readline.pyi

```diff
@@ -1,10 +1,11 @@
 import sys
 from _typeshed import StrOrBytesPath
-from typing import Callable, Optional, Sequence
+from typing import Callable, Sequence
 
 if sys.platform != "win32":
-    _CompleterT = Optional[Callable[[str, int], Optional[str]]]
-    _CompDispT = Optional[Callable[[str, Sequence[str], int], None]]
+    _CompleterT = Callable[[str, int], str | None]
+    _CompDispT = Callable[[str, Sequence[str], int], None]
```

Comment:
Nit: These shouldn't be named ending with `T`, because they aren't type variables.